lib/knapsack_pro/config/env.rb

```diff
@@ -186,7 +186,16 @@ def fixed_test_suite_split?
         end
 
         def fixed_queue_split
-          ENV.fetch('KNAPSACK_PRO_FIXED_QUEUE_SPLIT', false)
+          @fixed_queue_split ||= begin
+            env = ENV['KNAPSACK_PRO_FIXED_QUEUE_SPLIT']
+            computed = env || ci_env_for(:fixed_queue_split).to_s
```

Comment:
ENV value is always string so it's fine to use `env || ci_env_for(:fixed_queue_split).to_s` even when `KNAPSACK_PRO_FIXED_QUEUE_SPLIT=false` then the `false` string value is used instead of `ci_env_for(:fixed_queue_split).to_s`.

But if someone would stub KNAPSACK_PRO_FIXED_QUEUE_SPLIT to false as boolean in test environment then it would behave differently.

The more explicit approach would be:

```
ENV.fetch('KNAPSACK_PRO_FIXED_QUEUE_SPLIT', ci_env_for(:fixed_queue_split).to_s)
```

But I leave it up to you. The current code also works fine.

Reply:
You mean if somebody in the Knapsack team stubbed it, right? Fixed it.

Reply:
Yes.